src/web/components/PortalHeader/PortalHeader.tsx

```diff
@@ -34,14 +35,26 @@ export function PortalHeader({
   logout,
 }: Readonly<PortalHeaderProps>) {
   const { participantId } = useParams();
+  const parsedParticipantId = parseParticipantId(participantId);
+
   const { LoggedInUser } = useContext(CurrentUserContext);
+  const user = LoggedInUser?.user;
+  const userRolesForCurrentParticipant = user?.participants?.find(
+    (p) => p.id === parsedParticipantId
+  )?.currentUserRoleIds;
 
   const routes = [ParticipantInformationRoute, TeamMembersRoute, EmailContactsRoute];
-  if (LoggedInUser?.user?.isUid2Support) {
+
+  if (
+    user?.isUid2Support ||
+    [UserRoleId.UID2Support, UserRoleId.Admin].some((role) =>
```

Comment:
Is there any danger of `isUid2Support` not being populated properly? I guess we could technically just check `UserRoleId.Admin` here since we are checking `isUid2Support` in the line above